clean.cmd

```diff
@@ -103,4 +116,13 @@ echo     /t     - Deletes the tools directory.
 echo     /s     - Deletes the untracked files under src directory (git clean src -xdf).
 echo     /all   - Combines all of the above.
 echo.
-echo If no option is specified then clean.cmd /b is implied.
\ No newline at end of file
+echo If no option is specified then clean.cmd /b is implied.
+
+exit /b 1
```

Comment:
I decided to do that because that's what build.sh does.  I'm not sure if it's an established general pattern, but I thought it made sense to be consistent with build.sh.